src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/UpdateAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -78,7 +78,8 @@ public class SetAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
             HelpMessage = "A list of P2S VPN client tunneling protocols")]
         [ValidateSet(
             MNM.VpnClientProtocol.SSTP,
-            MNM.VpnClientProtocol.IkeV2)]
+            MNM.VpnClientProtocol.IkeV2,
```

Comment:
@henry416 
@markcowl had the following comment in the related PR : "Change this from a ValidateSet to an ArgumentCompleter" . Kindly address it.

Reply:
@praries880 Will not change, as described in the previous PR. This breaks the existing powershell inputs.